rotkehlchen/chain/gnosis/modules/airdrops/decoder.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from typing import TYPE_CHECKING, Any
+
+from rotkehlchen.chain.ethereum.modules.airdrops.constants import CPT_COW
+from rotkehlchen.chain.evm.decoding.airdrops import decode_cow_claim
+from rotkehlchen.chain.evm.decoding.interfaces import DecoderInterface
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import A_GNOSIS_VCOW
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.types import ChecksumEvmAddress
+    from rotkehlchen.user_messages import MessagesAggregator
+
+COW_PROTOCOL = string_to_evm_address('0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB')
```

Comment:
again with this. You are repeating constants across file. And as I told you this is not needed. You have this address.
```suggestion
```